src/box.tsx

```diff
@@ -14,6 +15,35 @@ const Box: BoxComponent = ({ is = 'div', innerRef, children, ...props }) => {
     parsedProps.ref = innerRef
   }
 
+  /**
+   * If the user has enabled safe hrefs we want to make sure that the url passed
+   * is safe and that the other attributes that make the link safe are added to the
+   * element
+   */
+  if (getUseSafeHref() && is === 'a' && parsedProps.href) {
```

Comment:
This whole block could be probably be extracted and given a name so its semantics are clearer.

Reply:
Makes sense to me